### ISSUE TITLE
Disables mulitple critical tools: Logcat, Running Devices, Device Manager

### ISSUE THREAD

**Issue by funkyidol** [crowdin/android-studio-plugin#130]
**Describe the bug**
Enabling and restarting the Android Studio disables many critical tools in Android Studio:
1. Logcat
2. Running Devices
3. Device Manager

**To Reproduce**
1. Configuration file: 'config.xml'
```files:
  - source: "app/src/main/res/values/strings.xml"
    translation: "app/src/main/res/values-%android_code%/strings.xml"
```
4. Localization files structure: Standard Android
5. Install Crowdin Jetbrains plugin from marketplace
6. Restart Android Studio

**Environment:**
1. Android Studio version: Android Studio Ladybug Feature Drop | 2024.2.2 Canary 1
Build #AI-242.21829.142.2422.12329062, built on September 6, 2024
Runtime version: 21.0.3+-79915917-b509.11 aarch64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Toolkit: sun.lwawt.macosx.LWCToolkit
macOS 14.6.1
GC: G1 Young Generation, G1 Concurrent GC, G1 Old Generation
Memory: 5120M
Cores: 12
Metal Rendering is ON
Registry:
  ide.instant.shutdown=false
  ide.balloon.shadow.size=15
  debugger.new.tool.window.layout=true
  ide.experimental.ui=true
  i18n.locale=
  ide.images.show.chessboard=true
  terminal.new.ui=true
Non-Bundled Plugins:
  com.markskelton.one-dark-theme (5.11.0)
  com.godwin.json.parser (1.7.1)
  com.facebook.ktfmt_idea_plugin (1.2.0.52)
  Key Promoter X (2024.2.0)
  some.awesome (1.15)
  wu.seal.tool.jsontokotlin (3.7.4)
  com.jetbrains.edu (2024.8-2024.2-749)
  Gitflow-Fix (0.7.13)
  mobi.hsz.idea.gitignore (4.5.3)
  com.crowdin.crowdin-idea (2.0.2)

3. OS version: MacOS Sonoma 14.6.1

**Expected behavior**

**Screenshots**
Before
![Screenshot 2024-09-10 at 9 50 57 PM](https://github.com/user-attachments/assets/e07d4a21-d7e1-47dd-aaa4-67c6f7a1ec21)

After
![Screenshot 2024-09-10 at 9 51 33 PM](https://github.com/user-attachments/assets/c2df1144-136f-4861-9b3e-150a63df72c5)

Observe the bottom left icons and top right icons missing

**Additional context**

Run the plugin in Debug mode and provide an extended log.
To enable Debug mode add `debug=true` to the *crowdin.properties* file.

**Comment by yevheniyJ** [comment]
@funkyidol This bug is not reproducible for me on latest stable build, which is `2024.1.2`. On `2024.2.2` it is indeed reproducible.
Since it's a canary build which means that it's not fully stable, I would recommend to switch to stable build.
Can you please try it?

**Comment by funkyidol** [comment]
Sure let me give it a shot and get back to you

**Comment by yevheniyJ** [comment]
@funkyidol Any updates on this one?

**Comment by funkyidol** [comment]
I can confirm that this is not as issue on the following version of AS

Android Studio Ladybug | 2024.2.1 Beta 2
Build #AI-242.21829.142.2421.12332550, built on September 6, 2024
Runtime version: 21.0.3+-79915917-b509.11 aarch64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Toolkit: sun.lwawt.macosx.LWCToolkit
macOS 14.7
GC: G1 Young Generation, G1 Concurrent GC, G1 Old Generation
Memory: 5120M
Cores: 12
Metal Rendering is ON
Registry:
  ide.instant.shutdown=false
  ide.balloon.shadow.size=15
  debugger.new.tool.window.layout=true
  ide.experimental.ui=true
  i18n.locale=
  ide.images.show.chessboard=true
  terminal.new.ui=true
Non-Bundled Plugins:
  com.markskelton.one-dark-theme (5.12.0)
  com.godwin.json.parser (1.7.1)
  com.facebook.ktfmt_idea_plugin (1.2.0.52)
  Key Promoter X (2024.2.0)
  some.awesome (1.15)
  wu.seal.tool.jsontokotlin (3.7.5)
  com.jetbrains.edu (2024.8-2024.2-749)
  Rider UI Theme Pack (0.15.0)
  Gitflow-Fix (0.7.13)
  mobi.hsz.idea.gitignore (4.5.3)
  com.crowdin.crowdin-idea (2.0.2)

**Comment by yevheniyJ** [comment]
@funkyidol Thank you! In this case I will close this issue. And reopen it only if it will appear again on stable AS version
